src/test/codegen/async-fn-debug.rs

```diff
@@ -50,7 +50,7 @@ async fn async_fn_test() {
 // CHECK:      {{!.*}} = !DIDerivedType(tag: DW_TAG_member, name: "s", scope: [[S1]]
 // CHECK-NOT:  flags: DIFlagArtificial
 // CHECK-SAME: )
-// CHECK:      [[DISC]] = !DIDerivedType(tag: DW_TAG_member, name: "__state", scope: [[ASYNC_FN]],
+// CHECK:      [[DISC]] = !DIDerivedType(tag: DW_TAG_member, name: "__state", scope: [[GEN]],
```

Comment:
The DW_TAG_member describing the discriminant of the DW_TAG_variant_part is now moved into the top-level DW_TAG_struct_type as well, instead of being a sibling. This is still wrong, because it is actually a child of the DW_TAG_variant_part, but the LLVM APIs don't let us do it completely correctly.